### DISCUSSION TITLE
QQ9.0.90装上后多次打开出现闪退

### DISCUSSION THREAD

**Discussion by Shiroha1768** [cinit/QAuxiliary#1184] [Q&A]
![Screenshot_2024-09-14-12-58-49-40.jpg](https://github.com/user-attachments/assets/90151a56-c964-4613-9aa3-e1e305877ba8)

**Comment by cinit** [comment]
请提供崩溃日志（可以使用 adb 抓取）

**Reply by Shiroha1768** [reply]
https://blog.csdn.net/Python_0011/article/details/132040387
没操作过，按这篇文章操作对吗？

**Reply by cinit** [reply]
打开 qa 的 “禁用 QQCrashReportManager”, 等待至少出现两次闪退，使用 adb 连接手机，adb 连上后把 /data/tombstones/ 里的东西取出来，应该这里会有崩溃日志。

**Reply by Shiroha1768** [reply]
![image](https://github.com/user-attachments/assets/c1d97f46-c523-4f1f-a5b6-68d4d39ebb60)
提取/data里的文件手机要root,我没有root权限不够，我用chatGPT复制了logcat命令提取了日志，大佬看一下，这样可以吗？
![image](https://github.com/user-attachments/assets/3796c127-84de-40bc-a3a0-98da6dab44db)
[logcat_output.zip](https://github.com/user-attachments/files/17001393/logcat_output.zip)

**Reply by cinit** [reply]
这个问题是“屏蔽更新”功能导致的，请关闭qa的“屏蔽更新”功能。

**Reply by Shiroha1768** [reply]
> 这个问题是“屏蔽更新”功能导致的，请关闭qa的“屏蔽更新”功能。

好的，谢谢大佬😊
